array.txt

```diff
@@ -0,0 +1,23 @@
+arr=[1,2,2,3,4,5,8,9,12]
+puts "enter the number"
+m=gets.chomp.to_i
+arr.sort!
+b=arr.combination(2).to_a

```

Comment:
can you use without combination method?
so it can have understand logic behind in it